src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionxpathfilter/InputSuppressionXpathFilterCyclomatic.java

```diff
@@ -0,0 +1,11 @@
+package com.puppycrawl.tools.checkstyle.filters.suppressionxpathfilter;
+
+public class InputSuppressionXpathFilterCyclomatic {
```

Comment:
it might be related to my above question about single violation in file....

if we go this way, we will definitely need several test inputs for each check, to test its ability to suppress different types of violations(on different AST elements), so we need to create extra folder for each check under "suppressionxpathfilter".